ansible/vars/epoch/default.yml

```diff
@@ -1,6 +1,8 @@
 api_base_uri: http://{{ public_ipv4 }}:{{ epoch_config.http.external.port }}/v2
 genesis_accounts:
   "ak_UAzhn9rAQg568v6Hwt3w2HPaQb9X9Nw6JbLmnv7trhmGmWGGp": 100000000001
+configure_peers: true
```

Comment:
Shall [`configure_peers|default(true)`](https://github.com/aeternity/infrastructure/blob/c2326292a5c04801136e7916c61f584bdab531c0/ansible/templates/epoch.yaml#L2) be modified to check that `configure_peers` is mandatory, though without default `true` or `false` value, so to prevent typos? (It seems always specified now.) Totally optional of course.

Reply:
I would like to refactor this variable name in future, maybe I'll rethink it's defaults and it should be mandatory or not then